core/src/main/java/hudson/security/Permission.java

```diff
@@ -340,7 +354,7 @@ public boolean getEnabled() {
     /**
      * Generic read access.
      */
-    public static final Permission READ = new Permission(GROUP,"GenericRead",null,HUDSON_ADMINISTER);
+    public static final Permission READ = new Permission(GROUP,"GenericRead",null,CONFIGURE_JENKINS);
```

Comment:
Originally (before FGP) this READ permission was implied directly by ADMINISTER.

Also, the existent CONFIGURE allows configuring items (like agents, jobs, etc). We don't want CONFIGURE_JENKINS to allow that, so can't imply it.

Couldn't find a better way of making CONFIGURE_JENKINS imply Jenkins.READ and Permission.READ.